app/views/admin/users/index.haml

```diff
@@ -0,0 +1,22 @@
+.users
+  %table.users
+    %tr
+      %th Name
+      %th.roles Admin
```

Comment:
Seems like applying CSS based on an identifier for the table would be simpler than attaching the same identifier to nearly every th and td.